app/views/trainees/forbidden_deletes/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<%= render PageTitle::View.new(i18n_key: "trainees.forbidden_deletes.show",
+                               has_errors: @trainee_forbidden_deletes_form.errors.present?) %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render DynamicBackLink::View.new(@trainee, text: t(:back)) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <%= register_form_with(model: @trainee_forbidden_deletes_form,
+                           url: trainee_forbidden_deletes_path(@trainee),
+                           local: true) do |f| %>
+      <%= f.govuk_error_summary %>
+      <%= f.hidden_field :alternative_option %>
+
+      <%= render TraineeName::View.new(@trainee) %>
+
+      <h1 class="govuk-heading-l"><%= t(".heading") %></h1>
+      <p class="govuk-body"><%= t(".forbidden_reason") %></p>
```

Comment:
The _Forbidden Reason_ 🍎

Reply:
I don't know why but this naming made me chuckle :)

Reply:
I struggled with this. "Forbidden" was the best word that gelled with "Delete". But, yeah it's got that forbidden ruit vibe to it. :-)

Reply:
I might go with `reason` instead.